clusters/sqnc-staging/base/flux-system/gotk-sync.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: main
+    branch: chore/upgrade_org_data_attachment_service
```

Comment:
Don't forget to suspend the git source, change this to main, merge and then resume the git source.